changelog/7760.breaking.rst

```diff
@@ -0,0 +1,2 @@
+`sunpy.map.GenericMap.contour` now uses ContourPy instead of scikit-image for faster performance and consistency with `sunpy.map.GenericMap.draw_contour`, reducing reliance on scikit-image.
```

Comment:
```suggestion
`sunpy.map.GenericMap.contour` now uses ContourPy instead of scikit-image for faster performance and consistency with `sunpy.map.GenericMap.draw_contour`.
```
Is it actually faster? We can't say this if we don't verify it.

Reply:
Alright, how about 'improved' instead of 'faster' as suggested earlier

Reply:
Improved how?

Reply:
I don't know, I tried looking for evidence but didn't find any, so what should I update changelog to?

Reply:
Either you drop the mention of performance or you benchmark it. That is our two choices.

Reply:
**Performance Test**
```
import cProfile

test_code = '''

import matplotlib.pyplot as plt

import astropy.units as u
for x in range(25):
    import sunpy.map
    from sunpy.data.sample import AIA_193_IMAGE
    aiamap = sunpy.map.Map(AIA_193_IMAGE)
    contours = aiamap.contour(50000 * u.DN)
'''

if __name__ == "__main__":
    cProfile.run(test_code)
```
**Using Scikit-Image**
`8894305 function calls (18373215 primitive calls) in 62.470 seconds`
**Using ContourPy**
`17448686 function calls (16975009 primitive calls) in 58.905 seconds`

Reply:
That is better. Improved it is.